subsys/event_manager/event_manager.c

```diff
@@ -14,8 +14,7 @@
 #ifdef CONFIG_SHELL
 extern u32_t event_manager_displayed_events;
```

Comment:
when it is extern is it set to something in the other file?

Reply:
It is initialized in this file (using the same funciton no matter if shell is in use or not)